runner/logs.go

```diff
@@ -538,12 +538,7 @@ func testOtelLog(ctx context.Context, wg *sync.WaitGroup, server *state.Server,
 	logTestSucceeded := make(chan bool, 1)
 	parsedLogStream := setupLogStreamer(ctx, wg, opts, logger, []*state.Server{server}, logTestSucceeded, stream.LogTestCollectorIdentify)
 
-	err := selfhosted.SetupOtelHandlerForServer(ctx, wg, opts, logger, server, parsedLogStream)
```

Comment:
memo: I was wondering why this error was removed, but looks like this error was never really used (error was never really returned) anyways, so should be no effect of removing